Cubical/Structures/Free.agda

```diff
@@ -37,19 +37,66 @@ module Definition {f a e n s : Level} (σ : Sig f a) (τ : EqSig e (ℓ-max n s)
     ext : {X : Type ℓ} {𝔜 : struct (ℓ-max ℓ' ns) σ}
           (H : isOfHLevel h (𝔜 .car)) (ϕ : 𝔜 ⊨ ε)
        -> (hom : X -> 𝔜 .car) -> structHom < F X , α > 𝔜
-    ext h ϕ = invIsEq (isFree h ϕ)
+    ext H ϕ = invIsEq (isFree H ϕ)
```

Comment:
Is it possible to rename `ext` to `_#`?

Reply:
Yeah I tried but couldn't figure out how, made a write up here https://github.com/pufferffish/agda-symmetries/issues/33#issuecomment-1756535299